packages/web/src/components/search/SearchBox.js

```diff
@@ -1118,7 +1176,39 @@ const SearchBox = (props) => {
 		return '/';
 	};
 
-	const renderIcons = () => {
+	const ThumbnailOrIcon = props.showImageSearch && currentImageValue ? (
```

Comment:
@SavvyShah  Can we memoize this?

Reply:
It's unnecessary here.

Reply:
@SavvyShah When you answer such review comments, I would expect a response with a rationale accompanied by your answer. We are open-sourced and I would suggest let's make it easy for the users who go through the PRs and reviews to gain something out of the comments we exchange on the platform.

Reply:
There's no performance improvement in using useMemo here. It's returning JSX element and the DOM won't be updated until the elements returned are different.